draft-ietf-dnsop-svcb-https.md

```diff
@@ -1082,18 +1082,32 @@ single-client granularity.
 
 ## Interaction with Alt-Svc
 
-Clients that do not implement support for Encrypted ClientHello MAY
-skip the HTTPS RR query
-if a usable Alt-Svc value is available in the local cache.
-If Alt-Svc connection fails, these clients SHOULD fall back to the HTTPS RR
-client connection procedure ({{client-behavior}}).
-
-Clients that implement support for ECH MUST perform the HTTPS RR query first,
-and MUST only make use of Alt-Svc when operating in SVCB-optional mode (see
-{{ech-client-behavior}}).
-
-This specification does not alter the DNS records used when connecting
-to an Alt-Svc hostname (typically A and/or AAAA only).
+Clients that implement support for both Alt-Svc and HTTPS records SHOULD
+retrieve any HTTPS records for the Alt-Svc alt-authority, and ensure that
+their connection attempts are consistent with both the Alt-Svc parameters
+and any received HTTPS SvcParams.  If present, the HTTPS record's target
+and port override the alt-authority.  For example, suppose that
+"https://example.com" sends an Alt-Svc field value of:
+
+    Alt-Svc: h2="alt.example.com:443", h3=":8443"
+
+The client would retrieve the following HTTPS records:
+    alt.example.com. IN HTTPS 1 . alpn=h2,h3 ech=...
```

Comment:
```suggestion

    alt.example.com. IN HTTPS 1 . alpn=h2,h3 ech=...
```
I think that you need vertical space here for this to render correctly.

Using `~~~` around figures is good hygiene and that allows you to [annotate](https://www.rfc-editor.org/materials/sourcecode-types.txt) them correctly.  (Like using `~~~http-message` for the example above.)